R/shiny.R

```diff
@@ -628,6 +628,15 @@ ShinySession <- R6Class(
       fun %||% identity
     },
 
+    # Ensure that `private$currentThemeDependency` is initialized. We don't do
+    # this in the `initialize()` method because then the reactiveVal doesn't
+    # have a reactive domain, and can persist after the app has exited.
```

Comment:
```suggestion
    # Used in getCurrentTheme()/setCurrentTheme() to ensure their reactive 
    # dependency is initialized. Note that we don't initialize this reactiveVal()
    # inside the initialize() method because at that point there isn't a
    # reactive domain yet, and so it can persist after the app has exited.
```